pkg/segment/reader/segread/segstatsreader.go

```diff
@@ -183,7 +197,49 @@ func readSingleSst(fdata []byte, qid uint64) (*structs.SegStats, error) {
 	}
 	sst.NumStats.Sum = sum
 
-	return &sst, nil
+	if readNumCount {
+		idx += 8
+
+		// read NumCount
+		sst.NumStats.NumCount = toputils.BytesToUint64LittleEndian(fdata[idx : idx+8])
+	}
+}
+
+func readCurrentSST(sst *structs.SegStats, fdata []byte, idx uint32) error {
```

Comment:
What does the `f` mean in `fdata`?

Reply:
fdata refers to file data.

Reply:
what does `current` mean ? is tied to the V4 or something else ? when does current become past ?